tests/member_chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,389 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`break-last-call.php 1`] = `
+<?php
+function call($store) {
+    return callApi($endpoint, $schema)->then(
+        function ($response) {
+            return next(actionWith([
+                'response' => $response,
+                'type' => $successType
+            ]));
+        },
+        function ($error) {
+            return next(actionWith([
+                'type' => $failureType,
+                'error' => $error->message || 'Something bad happened'
+            ]));
+        }
+  );
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+function call($store)
+{
+    return callApi($endpoint, $schema)->then(function ($response) {
+        return next(
+            actionWith(['response' => $response, 'type' => $successType])
+        );
+    }, function ($error) {
```

Comment:
Should we start with newlines when we have two function? But not related to this issue. Feel free create issue around this,

Reply:
https://github.com/prettier/plugin-php/issues/374